style/checkstyle.xml

```diff
@@ -271,9 +271,8 @@
         <module name="EmptyStatement" />
 
         <!-- Checks for Java Docs. -->
-        <module name="JavadocStyle">
-            <property name="severity" value="info"/>
-        </module>
+        <module name="JavadocStyle"/>
+
```

Comment:
The default severity is `error`, is the time to change it?

Reply:
Since the PR(#1081) does the similar thing.

Reply:
> The default severity is `error`, is the time to change it?

yes, I fix all codes based on JavadocStyle.

Reply:
> Since the PR(#1081) does the similar thing.

right